src/Sulu/Bundle/AdminBundle/Resources/js/components/MultiItemSelection/Header.js

```diff
@@ -23,6 +24,7 @@ export default class Header<T: string | number, U: string | number> extends Reac
 
     render() {
         const {
+            className,
```

Comment:
Why not making this more explicit, and pass a `disabled` prop instead? Then we also would not have the `header` class applied twice in the tests.

Reply:
And it would be consistent with the `ItemWrapper` below.

Reply:
fixed